static/src/javascripts/.eslintrc.js

```diff
@@ -31,7 +31,12 @@ module.exports = {
         'no-restricted-imports': [
             'error',
             {
-                paths: ['lodash'],
+                paths: [
+                    'lodash',
+                    'lodash/collections/forEach',
+                    'lodash/collections/map',
+                    'lodash/collections/reduce',
+                ],
```

Comment:
What do you think about adding 

- [ ] `collections/filter`
- [ ] `collections/some`
- [ ] `objects/assign`
- [ ] `objects/values`
- [ ] `objects/merge`
- [ ] `objects/keys`
- [ ] `collections/every`
- [ ] `collections/contains`
- [ ] `objects/isArray`
- [ ] `arrays/indexOf`

This would allow the removal a huge part of lodash ...